server/db-patches/patch-6-7.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE images DROP COLUMN image;
+ALTER TABLE images ADD COLUMN url TEXT;
```

Comment:
Doesn't need to happen in this patch, but I realized it would be nice to store the image sizes (bytes) in the database.